src/main/java/edu/cornell/kfs/concur/batch/service/impl/ConcurCashAdvancePdpFeedFileServiceImpl.java

```diff
@@ -81,7 +82,26 @@ else if (detailFileLine.getValidationResult().isCashAdvanceLine() && detailFileL
 
     private void recordCashAdvanceGenerationInDuplicateTrackingTable(ConcurRequestExtractRequestDetailFileLine detailFileLine, String soureDocumentNumber, String requestExtractFileName) {
         ConcurRequestedCashAdvance duplicateTrackingCashAdvance =
-                new ConcurRequestedCashAdvance(detailFileLine.getRequestId(), detailFileLine.getEmployeeId(), detailFileLine.getRequestAmount(), detailFileLine.getBatchDate(), soureDocumentNumber, requestExtractFileName);
+                new ConcurRequestedCashAdvance(detailFileLine.getRequestId(),
+                                               detailFileLine.getEmployeeId(),
+                                               detailFileLine.getRequestAmount(),
+                                               detailFileLine.getBatchDate(),
+                                               soureDocumentNumber,
+                                               detailFileLine.getCashAdvanceKey(),
+                                               detailFileLine.getChart(),
+                                               detailFileLine.getAccountNumber(),
+                                               StringUtils.defaultIfBlank(detailFileLine.getSubAccountNumber(), KFSConstants.EMPTY_STRING),
+                                               getConcurBatchUtilityService().getConcurParameterValue(ConcurParameterConstants.DEFAULT_TRAVEL_REQUEST_OBJECT_CODE),
+                                               StringUtils.defaultIfBlank(detailFileLine.getSubObjectCode(), ""),
+                                               StringUtils.defaultIfBlank(detailFileLine.getProjectCode(), ""),
+                                               StringUtils.defaultIfBlank(detailFileLine.getOrgRefId(), ""),
```

Comment:
I would suggest replacing the "" literals on these lines with KFSConstants.EMPTY_STRING or StringUtils.EMPTY, like what was done a few lines above this.